cg/store/crud/update.py

```diff
@@ -106,3 +109,12 @@ def update_analysis_upload_started_at(
         analysis = self.get_analysis_by_entry_id(analysis_id)
         analysis.upload_started_at = upload_started_at
         self.session.commit()
+
+    def update_order_type_applications(
+        self, application: Application, order_types: list[OrderType]
+    ):
+        """Updates the order_type_application table based on the updated Application object"""
+        self._get_query(OrderTypeApplication).filter_by(application_id=application.id).delete()
```

Comment:
are we not deleting stuff here? If so, that should be handled in the DeleteHandler.

Reply:
Yes, I have put way too much thought into where to put stuff. I think nothing should be in the update handler. I am refactoring

Reply:
Moved it to the ApplicationService